doc/example-requests/adapter.http

```diff
@@ -131,3 +131,45 @@ Content-Type: application/json
     "license": "AGPL3"
   }
 }
+
+### Performe Manual Data Import without runtime parameters
+POST {{baseURL}}/datasources/{{datasourceId}}/trigger HTTP/1.1
+Content-Type: application/json
+
+### Create datasource with dynamic parameters
+POST {{baseURL}}/datasources  HTTP/1.1
+Content-Type: application/json
+
+{
+  "protocol": {
+    "type": "HTTP",
+    "parameters": {
+      "location": "https://www.pegelonline.wsv.de/webservices/rest-api/v2/stations/{station}/W/measurements.json?start=P1D",
```

Comment:
Looks good!
Do we have to specify the parameters somewhere further? Like default value, etc.? Or is that an extra PR?

Reply:
That is coming in supplemental PR. The problem is that for default values I have to touch the model of the datasource. So I must probably touch a couple of files and it would get unnecessarily cluttered.